Integrate branches with correct pr attribution (#3)
timescale/pgai: DEVELOPMENT.md

- [Test your pgai environment](#test-your-pgai-environment): use the tests.sql script to validate your pgai environment.

## pgai Prerequisites

Before you start working with pgai, you need:

* An [OpenAI API Key](https://platform.openai.com/api-keys).
* [Psql](https://www.timescale.com/blog/how-to-install-psql-on-mac-ubuntu-debian-windows/) or [PopSQL](https://docs.timescale.com/use-timescale/latest/popsql/)
* The pgai source on your local machine:
   ```bash
   git clone git@github.com:timescale/pgai.git
   ```
* Your virtual environment, either:
    * [Docker](https://docs.docker.com/get-docker/)
    * [Multipass](https://multipass.run/)
    * Or both, why not :metal:? 

## Setup a pgai environment in Docker

The pgai Docker container has all the software you need preinstalled. To connect to
pgai running in a Docker container:

1. In Terminal, navigate to the folder you cloned this pgai repository to.

1. Build the Docker image:

   ```bash
   docker build -t pgai .
   ```

1. Run the container:

    ```bash
    docker run -d --name pgai -p 9876:5432 -e POSTGRES_PASSWORD=pgaipass --mount type=bind,src=`pwd`,dst=/pgai pgai
    ```

>    You are automatically connected to the container. To connect from the command line, run

Reviewer: ```suggestion
   To get a shell inside the container, run
```
The docker version does not automatically connect you to the container.
